### PR TITLE
Alchemical abilities

### PR DESCRIPTION
**Pull request by Adrail86**
# Submission Checklist
## Required

- [x] The pull request title clearly contains the name of the sheet I am editing.
- [x] The pull request title clearly states the type of change I am submitting (New Sheet/New Feature/Bugfix/etc.).
- [x] The pull request makes changes to files in only one sub-folder.
- [x] The pull request does not contain changes to any json files in the translations folder (translation.json is permitted)

## New Sheet Details

- The name of this game is: <Exalted>  _(i.e. Dungeons & Dragons 5th Edition, The Dresden Files RPG)_
- The publisher of this game is: <White Wolf Publishing> _(i.e. Wizards of the Coast, Evil Hat)_
- The name of this game system/family is: <Exalted 2e> _(i.e. Dungeons & Dragons, FATE)_

- [X] I have followed the [Character Sheets Standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) when building this sheet.

- [ ] This sheet has been made on behalf of, or by, the game's publisher.
- [ ] This game is not a traditionally published game, but a copy of the game rules can be purchased/downloaded/found at: <   >
- [X] This sheet is for an unofficial fan game, modification to an existing game, or a homebrew system.

# Changes / Description

<Added changes from recent Bug Fix by Briteblade and added alchemical ability tracking>

### CODE REVIEW

**Comment by Adrail86** [comment]
@Briteblade can you make sure that this change doesn't cause the same issue the last one did with formatting? I am still having trouble with the sandbox on roll20.

**Review by Adrail86 — COMMENTED** [review]
@Briteblade here is the Html code for the Alchemical ability addition

**Comment by Briteblade** [comment]
@Adrail86 Sure man, I'll look into it. It might take me a bit though, end of year stuff is kicking my ass. Should be able to take a look this weekend at the latest.

File: Exalted2e/exalted2e.css
Comment (Briteblade) [inline Exalted2e/exalted2e.css:1]: I believe we're missing code lines that are required here for the new Heartsblood tab to work.

File: Exalted2e/exalted2e.html [hunk 1]
```diff
@@ -4372,11 +4521,10 @@ <h3 class="ability">Specialties</h3>
...
                     </div>
-                </fieldset>
-            </div>
-        </div>       
-	</div>
- 
+                </fieldset>   
+    </div>    
+    </div>
+
```
Comment (Briteblade) [inline Exalted2e/exalted2e.html:4386]: Missing a </div> here to close the mortal ability sections

File: Exalted2e/exalted2e.html [hunk 2]
```diff
@@ -4579,8 +4727,483 @@ <h1 class="separator"><span>Experience</span></h1>
             </div>
         </div>
     </div>
+    <!-- Alchemicals / ability set 8-->
```
Comment (Briteblade) [inline Exalted2e/exalted2e.html:4589]: This entire ability section needs to be shifted up above the Advantages section

**Review by Briteblade — CHANGES_REQUESTED** [review]
Other than these changes here, everything looks like it should work. Just add that missing </div> and shift the Alchemical skills up to it's appropriate spot.

**Comment by NorWhal** [comment]
Hi @Adrail86! 

Unfortunately we are going to reject this Pull Request on the basis that all new sheets must meet the Minimum Requirements section of the [character sheet standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository).

Your sheet fails in the following ways:

- Using `<table>` for layout. Please use CSS flexboxes or grid instead

Many thanks!

**Comment by Briteblade** [comment]
Hey Nora,

This sheet was made well before Adrail or myself began working on it, and
before the rule against  <table>  was made, and thus had been grandfathered
in, as per the note in the linked character sheet standards.


>    - This is the most common reason for new sheets being rejected/delayed
>    by Roll20. There are old sheets in the repo using them, but they where
>    created before this rule against <table> was made.
>
>
 We're just attempting to maintain and update it. We can look into
rewriting to remove the <table> in the future, but in the meantime, could
you allow us to put this Pull Request through?

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by BronsonHall** [comment]
Hey @Briteblade & @Adrail86 ,
  Yep, you're correct, existing <table> elements aren't a blocker for updating existing sheets.  I've unblocked this PR, and am merging now.  

As one small note for the future, if you can include the name of the sheet being edited in the title of your PRs going forward (i.e. "[Exalted2e] Alchemical Abilities" or the like), that'd be greatly appreciated! 
  
Thank you both for your patience, and hard work maintaining the sheet!

**Comment by Adrail86** [comment]
The changes got reverted to before the previous pull request causing an error with the Advantages section of the sheet none of the changes made in this pull were included!

**Comment by BronsonHall** [comment]
@Adrail86 .

Ack - that's not good.  Yeah, I see the missing div tag from the earlier error.  If you can, please do make a new PR with that fix, tag or request me, and I'll make sure to get it merged as soon as I can.   Are you sure the changes in this PR didn't make it in alongside the error?  I'm seeing them in code on master - I think all you may need to do is restore that div tag?

**Pull request merged**